src/js/common/schemaform/components/AskVAQuestions.jsx

```diff
@@ -7,6 +7,12 @@ function AskVAQuestions(props) {
         <div className="help-footer-box">
           <h2 className="help-heading">Need help?</h2>
           {props.children}
+          <p className="help-talk">
```

Comment:
Adding this here will affect other forms. You can make a component for this content and use the `getHelp` property in the form config to pass this in and render it in line 9.